src/averageAges.js

```diff
@@ -36,8 +36,19 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter((woman) => woman.sex === 'f'
+    && (withChildren === undefined
```

Comment:
Don't compare with` undefined`/`null` or empty string. Use the logical not to check the falsy values.